app/controllers/contests_controller.rb

```diff
@@ -285,6 +285,18 @@ def supervise
       end
       redirect_to contestants_contest_path(@contest), :notice => "Contest ended for selected students"
     elsif params[:update]
+      if params[:school_id]
```

Comment:
Cool, but do we need this feature? If we select their username from the contestant list, then click edit, we can edit their school that way.

Reply:
It's slightly different, this is editing the school of the contest relation, whereas the other one is editing the school of the user. They're stored separately because a user might change schools after participating in a contest, and we want to preserve whichever school they were in at the time of the contest.

Reply:
Ah, okay. Good to know